rosplan_rqt/src/rosplan_rqt/ROSPlanDispatcher.py

```diff
@@ -6,7 +6,7 @@
 import sys
 
 from itertools import product
-from string import join, split
+# from string import join, split
```

Comment:
Could you please remove this line instead of commenting it out?

Reply:
Comment removed now.